.circleci/dependencies.py

```diff
@@ -34,6 +35,7 @@
     "cython": "0.29.32",
     "daphne": "4.0.0",
     "databases": "0.6.2",
+    "datadog_lambda": "4.66.0",
```

Comment:
I think the [library](https://pypi.org/project/datadog-lambda/) uses a hyphen rather than an underscore. Not sure if this matters in this context.

Reply:
you're right, thank you, I got mislead by the package import being an underscore, it didn't raise any complains, but I understand this should be updated

Reply:
It turns out it's an alias, but I guess we should be using the registered one.